### PR TITLE
bayesian quadrature block for gh-pages

### PR DESCRIPTION
**Pull request by mmahsereci**
*Issue #, if available:*

*Description of changes:*
I added a block for Bayesian Quadrature. Not sure if this is the style we are going for. 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: _posts/2016-06-09-bayesian-quadrature.markdown [hunk 1]
```diff
@@ -2,4 +2,56 @@
...
+problem on the integrand function ii) replacing the actual integration with an easier, analytic integration of the surrogate model
+on the integrand function, and iii) actively choosing locations for integrand evaluations such that the budget is optimally used
+in the sense encoded by the acquisition scheme.
+
+#### Bayesian Quadrature in Emukit
+Emukit provides basic functionality for vanilla Bayesian quadrature [[1, 2]](#refereces-on-quadrature), 
+but also for more elaborate methods like WSABI [[3]](#refereces-on-quadrature). It alleviated the need to research and 
+implement the integration of the emulator, the acquisition scheme and the active loop while providing a frame to implement 
+and try out novel Bayesian quadrature methods.
+
```
Comment (javiergonzalezh) [inline _posts/2016-06-09-bayesian-quadrature.markdown]: Once we have the code to run it, we can incorporate here an example of how to run the loop with Emukit.

File: _posts/2016-06-09-bayesian-quadrature.markdown [hunk 2]
```diff
@@ -2,4 +2,56 @@
...
+
+We're always open to contributions! Please read our [contribution guidelines](CONTRIBUTING.md) for more information. 
+We are particularly interested in contributions regarding translations and tutorials. 
+This project is licensed under Apache 2.0. Please refer to [LICENSE](LICENSE) and [NOTICE](NOTICE) for further license
```
Comment (javiergonzalezh) [inline _posts/2016-06-09-bayesian-quadrature.markdown]: I think we can remove the Apache bit here.

File: _posts/2016-06-09-bayesian-quadrature.markdown [hunk 3]
```diff
@@ -2,4 +2,56 @@
 layout: post
 title:  "Bayesian Quadrature"
 categories: jekyll update
+img: bayesian_quadrature.png
```
Comment (javiergonzalezh) [inline _posts/2016-06-09-bayesian-quadrature.markdown]: The figure is nice but I am not sure it encapsulates the idea of BQ, specially the subfigure on the right. I leave to you is you like it though.

File: _posts/2016-06-09-bayesian-quadrature.markdown [hunk 4]
```diff
@@ -2,4 +2,54 @@
...
 ---
+
+Bayesian quadrature [[1, 2]](#refereces-on-quadrature) is an active learning method for the value of an integral given queries of the integrand 
+on a finite and usually small amount of input locations.
+
+Bayesian quadrature returns not only an estimator, but a full posterior distribution on the integral value
+which can subsequently be used in decision making or uncertainty analysis.
+
+Bayesian quadrature is especially useful when integrand evaluations are expensive and sampling schemes 
+prohibitive, for example when evaluating the integrand involves running a complex computer simulation, a real-world experiments,
```
Comment (marpulli) [inline _posts/2016-06-09-bayesian-quadrature.markdown]:
```suggestion
prohibitive, for example when evaluating the integrand involves running a complex computer simulation, a real-world experiment,
```

File: _posts/2016-06-09-bayesian-quadrature.markdown [hunk 5]
```diff
@@ -2,4 +2,54 @@
...
+or a lab. But even when evaluation cost is manageable, the sheer amount of queries that might be required by classical 
+algorithms is usually incentive enough to favor a smarter and less wasteful approach.
+
+Instead, Bayesian quadrature draws information from structure encoded as prior over the function space modeling the integrand, 
+such as regularity or smoothness, and is thus able to learn faster from less datapoints (function evaluations).
```
Comment (marpulli) [inline _posts/2016-06-09-bayesian-quadrature.markdown]:
```suggestion
such as regularity or smoothness, and is thus able to learn faster from fewer datapoints (function evaluations). 
```

File: _posts/2016-06-09-bayesian-quadrature.markdown [hunk 6]
```diff
@@ -2,4 +2,54 @@
...
+
+Instead, Bayesian quadrature draws information from structure encoded as prior over the function space modeling the integrand, 
+such as regularity or smoothness, and is thus able to learn faster from less datapoints (function evaluations). 
+In Emukit, the surrogate model for the integrand would be the *emulator* and the integrand that can be queried, and
+ possibly describes a real, complex system, the *simulator*.
+
+#### Bayesian Quadrature in the Loop
+
+Like other sequential learning schemes, Bayesian quadrature iteratively selects points where the integrand will be queried 
+next such that and acquisition function (e.g., the reduction of the integral variance) is maximized in each step.
```
Comment (marpulli) [inline _posts/2016-06-09-bayesian-quadrature.markdown]:
```suggestion
next such that an acquisition function (e.g., the reduction of the integral variance) is maximized in each step. 
```

File: _posts/2016-06-09-bayesian-quadrature.markdown [hunk 7]
```diff
@@ -2,4 +2,54 @@
...
+
+#### Bayesian Quadrature in the Loop
+
+Like other sequential learning schemes, Bayesian quadrature iteratively selects points where the integrand will be queried 
+next such that and acquisition function (e.g., the reduction of the integral variance) is maximized in each step. 
+At the same time, the model class (often a Gaussian process) is updated with the newly collected datapoint and 
+refined at each step by optimizing its hyperparameters. 
+The actual integration that yields the distribution over the integral value is then performed by integrating the emulator
+of the integrand function, which is often analytic for certain choices of Gaussian processes, or 
+an analytic approximating (e.g., for WSABI [[3]](#refereces-on-quadrature)).
```
Comment (marpulli) [inline _posts/2016-06-09-bayesian-quadrature.markdown]: Do you mean "approximation" for the last word in this sentence?

File: _posts/2016-06-09-bayesian-quadrature.markdown [hunk 8]
```diff
@@ -2,4 +2,54 @@
...
+
+Like other sequential learning schemes, Bayesian quadrature iteratively selects points where the integrand will be queried 
+next such that and acquisition function (e.g., the reduction of the integral variance) is maximized in each step. 
+At the same time, the model class (often a Gaussian process) is updated with the newly collected datapoint and 
+refined at each step by optimizing its hyperparameters. 
+The actual integration that yields the distribution over the integral value is then performed by integrating the emulator
+of the integrand function, which is often analytic for certain choices of Gaussian processes, or 
+an analytic approximating (e.g., for WSABI [[3]](#refereces-on-quadrature)). 
+
+Thus, the success of Bayesian quadrature is based on three things: i) replacing an intractable integral with and inference
```
Comment (marpulli) [inline _posts/2016-06-09-bayesian-quadrature.markdown]:
```suggestion
Thus, the success of Bayesian quadrature is based on three things: i) replacing an intractable integral with an inference 
```

File: _posts/2016-06-09-bayesian-quadrature.markdown [hunk 9]
```diff
@@ -2,4 +2,54 @@
...
+an analytic approximating (e.g., for WSABI [[3]](#refereces-on-quadrature)). 
+
+Thus, the success of Bayesian quadrature is based on three things: i) replacing an intractable integral with and inference 
+problem on the integrand function ii) replacing the actual integration with an easier, analytic integration of the surrogate model
+on the integrand function, and iii) actively choosing locations for integrand evaluations such that the budget is optimally used
+in the sense encoded by the acquisition scheme.
+
+#### Bayesian Quadrature in Emukit
+Emukit provides basic functionality for vanilla Bayesian quadrature [[1, 2]](#refereces-on-quadrature), 
+but also for more elaborate methods like WSABI [[3]](#refereces-on-quadrature). It alleviated the need to research and
```
Comment (marpulli) [inline _posts/2016-06-09-bayesian-quadrature.markdown]:
```suggestion
but also for more elaborate methods like WSABI [[3]](#refereces-on-quadrature). It alleviates the need to research and 
```

File: _posts/2016-06-09-bayesian-quadrature.markdown [hunk 10]
```diff
@@ -2,4 +2,54 @@
...
+
+#### Bayesian Quadrature in Emukit
+Emukit provides basic functionality for vanilla Bayesian quadrature [[1, 2]](#refereces-on-quadrature), 
+but also for more elaborate methods like WSABI [[3]](#refereces-on-quadrature). It alleviated the need to research and 
+implement the integration of the emulator, the acquisition scheme and the active loop while providing a frame to implement 
+and try out novel Bayesian quadrature methods.
+
+
+We're always open to contributions! Please read our [contribution guidelines](CONTRIBUTING.md) for more information. 
+We are particularly interested in contributions regarding translations and tutorials.
```
Comment (marpulli) [inline _posts/2016-06-09-bayesian-quadrature.markdown]: Does translations mean implementation of existing methods?

File: _posts/2016-06-09-bayesian-quadrature.markdown (hunk 6 above)
Comment (mmahsereci) [reply _posts/2016-06-09-bayesian-quadrature.markdown]: done

File: _posts/2016-06-09-bayesian-quadrature.markdown (hunk 7 above)
Comment (mmahsereci) [reply _posts/2016-06-09-bayesian-quadrature.markdown]: yes, I do.

File: _posts/2016-06-09-bayesian-quadrature.markdown (hunk 8 above)
Comment (mmahsereci) [reply _posts/2016-06-09-bayesian-quadrature.markdown]: done

File: _posts/2016-06-09-bayesian-quadrature.markdown (hunk 9 above)
Comment (mmahsereci) [reply _posts/2016-06-09-bayesian-quadrature.markdown]: done

File: _posts/2016-06-09-bayesian-quadrature.markdown (hunk 10 above)
Comment (mmahsereci) [reply _posts/2016-06-09-bayesian-quadrature.markdown]: that is a good question. I just copied this over from the playground post without reading it properly

File: _posts/2016-06-09-bayesian-quadrature.markdown (hunk 3 above)
Comment (javiergonzalezh) [reply _posts/2016-06-09-bayesian-quadrature.markdown]: Something like the figure on the left but with a cleared pattern of points capturing the density would be nice. Something on the lines of the figure used in paper 1 here:

https://warwick.ac.uk/fac/sci/statistics/staff/research_students/briol/probabilistic_integration/

Comment (mmahsereci) [reply _posts/2016-06-09-bayesian-quadrature.markdown]: sounds good. I'll do another PR for this which just replaces the figure.

I'll need a running version of BQ for it and we don't have one right now.

**Pull request merged**